torch/_torch_docs.py

```diff
@@ -13897,3 +13897,82 @@ def merge_dicts(*dicts):
 are freshly created instead of aliasing the input.
 """,
 )
+
+
+for unary_base_func_name in (
+    "exp",
+    "sqrt",
+    "abs",
+    "acos",
+    "asin",
+    "atan",
+    "ceil",
+    "cos",
+    "cosh",
+    "erf",
+    "erfc",
+    "expm1",
+    "floor",
+    "log",
+    "log10",
+    "log1p",
+    "log2",
+    "neg",
+    "tan",
+    "tanh",
+    "sin",
+    "sinh",
+    "round",
+    "lgamma",
+    "frac",
+    "reciprocal",
+    "sigmoid",
+    "trunc",
+):
+    unary_foreach_func_name = f"_foreach_{unary_base_func_name}"
+    add_docstr(
+        getattr(torch, unary_foreach_func_name),
+        r"""
+{}(self: List[Tensor]) -> List[Tensor]
+
+Apply :func:`torch.{}` to each Tensor of the input list.
+
+If the input tensors meet a certain condition, for example, they have the same dtype,
+are hosted on the same cuda device, and contiguous (including channels_last and channels_last_3d),
+then the operation done with fewer cuda kernel calls.
+
+.. note::
+    Only CPU and CUDA are supported. Forward-mode AD is not supported.
+
+Args:
```

Comment:
These args are not correct for functions that take more arguments (like round). Should we just skip it altogether here and just recommend to read the original doc where all Tensor input/output are replaced with list of Tensors?

Reply:
That sounds right but I'm not aware of any unary foreach function taking optional arguments, let me see